packages/wp-now/esbuild.mjs

```diff
@@ -0,0 +1,10 @@
+import * as esbuild from 'esbuild';
+
+await esbuild.build({
+	entryPoints: ['packages/wp-now/src/main.ts'],
+	outdir: 'dist/packages/wp-now',
+	bundle: true,
+	packages: 'external',
+	format: 'esm',
+	platform: 'node',
+});
```

Comment:
is this not already supported in the `nx`/`vite` config syntax? if this were a config file I would imagine someone could run `esbuild --config wp-now.esbuild.json` and build the project without running through `nx`

I guess technically they could run `node esbuild.mjs`, but that's another level of discovery they have to learn they need to know about

Reply:
Yes!, it's possible to build the project without `nx` using only esbuild. @adamziel helped me set up this project configuration to make sure we could compile `wp-now` for node and not depend on compiling the rest of the projects from scratch for node.
Using `vite` or `nx` was unsuccessful because we fell into multiple source code corrections and dependencies that are not fully mandatory to fix, since it works with esbuild.

@dmsnell, I'll appreciate any help with this configuration and discover better ways to improve it.